Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -101,9 +100,9 @@
   - type: ExplodeOnTrigger
   - type: Explosive
     explosionType: Minibomb
-    totalIntensity: 200
-    intensitySlope: 30 #Will destroy the tile under it reliably, space 1-2 more to rods. Only does any significant damage in a 5-tile cross.
-    maxIntensity: 60
+    totalIntensity: 700
+    intensitySlope: 16 #If it's any lower then it feels really weak. If you're nerfing the minibomb, probably don't change just this value!
+    maxIntensity: 295 
```

Comment:
This is also going to effect the minibomb implant. Please revert it back and lower the timer to 3-4 seconds.
If you want a larger bomb with a long timer, add the macrobomb (with bomb diffusal) instead.

Reply:
Hm... I'm not sure what to think. The macrobomb explosion is _gigantic_ - nearly on the level of the syndicate bomb from SS13. I would be fine with making another bomb prototype as to not change the minibomb implant (and probably renaming it to something like the "explosive implant,") but I feel like this kind of defeats the purpose of the PR.
This would just make it better at gibbing stuff, which would probably be best suited to something like X-4 if we're going with SS13 analouges.

Additionally, a macrobomb would entail either nerfing the hell out of the implant or just making the syndicate bomb (which is a different item in purpose and function entirely, and also not something I do as I can't code.)

Reply:
I meant making a separate syndie bomb/macrobomb item, not to replace the minibomb with it.